packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -1849,30 +1882,29 @@ describe('TokenRatesController', () => {
           }) => {
             const tokenAddress = '0x0000000000000000000000000000000000000001';
 
-            await expect(
-              async () =>
-                await callUpdateExchangeRatesMethod({
-                  allTokens: {
-                    [ChainId.mainnet]: {
-                      [defaultSelectedAddress]: [
-                        {
-                          address: tokenAddress,
-                          decimals: 18,
-                          symbol: 'TST',
-                          aggregators: [],
-                        },
-                      ],
+            const updateExchangeRates = await callUpdateExchangeRatesMethod({
+              allTokens: {
+                [ChainId.mainnet]: {
+                  [defaultSelectedAddress]: [
+                    {
+                      address: tokenAddress,
+                      decimals: 18,
+                      symbol: 'TST',
+                      aggregators: [],
                     },
-                  },
-                  chainId: ChainId.mainnet,
-                  controller,
-                  triggerTokensStateChange,
-                  triggerNetworkStateChange,
-                  method,
-                  nativeCurrency: 'ETH',
-                  selectedNetworkClientId: InfuraNetworkType.mainnet,
-                }),
-            ).rejects.toThrow('Failed to fetch');
+                  ],
+                },
+              },
+              chainId: ChainId.mainnet,
+              controller,
+              triggerTokensStateChange,
+              triggerNetworkStateChange,
+              method,
+              nativeCurrency: 'ETH',
+              selectedNetworkClientId: InfuraNetworkType.mainnet,
+            });
+
+            expect(updateExchangeRates).toBeUndefined();
```

Comment:
OK, nice to know that this test does not throw now, but instead returns undefined.